src/createMollieClient.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
I think it's better to make an exception on line level instead of the entire file. Case the rule does make sense right?